corehq/apps/hqwebapp/static/hqwebapp/js/select2_knockout_bindings.ko.js

```diff
@@ -77,7 +77,7 @@ hqDefine("hqwebapp/js/select2_knockout_bindings.ko", [
 
         self.update = function (element, valueAccessor, allBindings) {
             var $el = $(element),
-                newValue = ko.unwrap(allBindings().value) || $el.val();
+                newValue = ko.unwrap(allBindings().value);
```

Comment:
@orangejenny I made this change to resolve [this issue](https://dimagi-dev.atlassian.net/browse/QA-3573), and I do feel like this is how the binding should behave. If the caller wants to reset the selected value to null, they should be able to. The other two use cases of this binding are in the copy app form which functions fine with this change, and the [graph_configuration_modal](https://github.com/dimagi/commcare-hq/blob/5841610faffcfe5ac59486adb99b1b3251ec9158/corehq/apps/app_manager/templates/app_manager/partials/modules/graph_configuration_modal.html#L254) which I'm not too familiar with.

Just wanted to flag this as it broadens the scope of this PR a bit.

Reply:
Cool. This change looks reasonable, but can you click around in a graph configuration modal and verify things look okay? [This menu's case detail](https://staging.commcarehq.org/a/bosco/apps/view/0168bb256b4e45b396de87c2af4ae4ae/module/91b55ea224e2f2a6f936c8d00d7c8fdbe00dc5f3/#case-detail-screen-detail-config-tab) has two graphs set up.

Reply:
Clicked around in the graph configuration and looks good. I'd expect that this would be only cause an "issue" if the property were being cleared in JS which it isn't so that gives me more confidence that this is a safe change.